### DISCUSSION TITLE
[DONE] Team: aliase names for notebooks

### DISCUSSION THREAD

**Discussion by jone** [ParetoSecurity/pareto-mac#66] [Ideas]
Hi

We're starting to test the new team dashboard 🎉 Thank you at this point for your work!

I've noticed that the dashboard shows the hostname of the machine. We do not have any rules for hostnames at our company, so I'll probably not know who has configured which name. I also would not want to have to make rules such as using the name of the person, because I wouldn't want to broadcast my name on foreign wifis.

So it would be great if I could just add an alias name to each machine once I figured out who's machine that is, so that I do not have to track that in another place. 😉 

Cheers

**Comment by dz0ny** [comment]
We already have planned a better way to show the real owner of the device.  I also want to point out that the name is not the hostname, but rather the iCloud device name, the hostname is configured separately see https://support.apple.com/en-ie/guide/mac-help/mchlp2322/mac#mchl8c79215b

**Comment by dmurko** [comment]
I really like the idea of adding an alias for a device! Added internally for a discussion.

**Comment by zupo** [comment]
Here's how I suggest we do it:

* Make `Device Name` editable.
* Add a new field `Reported Device Name` that gets updated every time Pareto Security app sends a report.
* Upon initial creation of a new device in Dashboard, `Device Name` is populated by the `Reported Device Name` value.

@jone: sounds good?

**Reply by jone** [reply]
@zupo That's simple and solves the problem. Perfect 😊 👍

**Comment by zupo** [comment]
Done!

1. Click the edit icon next to device name:

<img width="319" alt="Screenshot 2021-11-16 at 09 42 39" src="https://user-images.githubusercontent.com/311580/141961524-dbb4a5cc-69e1-487c-bb08-5d895b31864a.png">

2. Set the new device name:

<img width="822" alt="Screenshot 2021-11-16 at 09 42 34" src="https://user-images.githubusercontent.com/311580/141961565-57c9cc97-3e4d-4983-b761-6be9ebc12beb.png">
